packages/cli/package.json

```diff
@@ -64,12 +64,12 @@
   "devDependencies": {
     "@types/ejs": "^3.1.1",
     "@types/glob": "^7.2.0",
-    "@types/node": "^17.0.34",
+    "@types/node": "^18.15.11",
     "@types/openurl": "^1.0.0",
     "@types/yargs": "^17.0.10",
     "tsup": "^6.7.0",
     "tsx": "^3.12.6",
-    "vitest": "^0.29.8"
+    "vitest": "0.30.1"
```

Comment:
running vitest outputs a message that suggests pinning vitest because it doesn't follow semver